### DISCUSSION TITLE
Analyzing PBMC scATAC-seq Using Signac Error

### DISCUSSION THREAD

**Discussion by sohumsharma484** [stuart-lab/signac#1810] [Q&A]
Hi. I am trying to follow the directions at https://stuartlab.org/signac/articles/pbmc_vignette to analyze ATAC-seq data. However, I am running into the error "Error: unable to find an inherited method for function 'mcols' for signature 'x = "NULL"'" and am not sure what the issue is. I am new to bioinformatics and R. Any guidance would be appreciated.

Code:
![image](https://github.com/user-attachments/assets/2a353e46-c3d7-4b25-b009-012db0dbb098)

Output:
![image](https://github.com/user-attachments/assets/d51460c5-b0d2-452b-abfa-20d4f958b380)

**Comment by timoast** [comment]
Please open an issue with your complete code and output of `sessionInfo()`
